Instable integration tests

From time to time, builds on CircleCI fail with the following error message:

```
Tests run: 9, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 130.231 sec <<< FAILURE! - in org.sahli.asciidoc.confluence.publisher.docker.DockerBasedPublishingIntegrationTest
publish_withOnlyMandatoryEnvVars_publishesDocumentationToConfluence(org.sahli.asciidoc.confluence.publisher.docker.DockerBasedPublishingIntegrationTest)  Time elapsed: 9.264 sec  <<< ERROR!
java.lang.reflect.UndeclaredThrowableException
	at org.sahli.asciidoc.confluence.publisher.docker.DockerBasedPublishingIntegrationTest.publishAndVerify(DockerBasedPublishingIntegrationTest.java:222)
	at org.sahli.asciidoc.confluence.publisher.docker.DockerBasedPublishingIntegrationTest.publish(DockerBasedPublishingIntegrationTest.java:208)
	at org.sahli.asciidoc.confluence.publisher.docker.DockerBasedPublishingIntegrationTest.deleteAllPages(DockerBasedPublishingIntegrationTest.java:50)
Caused by: java.lang.reflect.InvocationTargetException
	at org.sahli.asciidoc.confluence.publisher.docker.DockerBasedPublishingIntegrationTest.publishAndVerify(DockerBasedPublishingIntegrationTest.java:222)
	at org.sahli.asciidoc.confluence.publisher.docker.DockerBasedPublishingIntegrationTest.publish(DockerBasedPublishingIntegrationTest.java:208)
	at org.sahli.asciidoc.confluence.publisher.docker.DockerBasedPublishingIntegrationTest.deleteAllPages(DockerBasedPublishingIntegrationTest.java:50)
Caused by: com.github.dockerjava.api.exception.ConflictException: 
{"message":"Cannot kill container: 7062e287027dd4300391dba01498e98cd6c63c8cb91b59182e2fad5ab33eb913: Container 7062e287027dd4300391dba01498e98cd6c63c8cb91b59182e2fad5ab33eb913 is not running"}

	at org.sahli.asciidoc.confluence.publisher.docker.DockerBasedPublishingIntegrationTest.publishAndVerify(DockerBasedPublishingIntegrationTest.java:222)
	at org.sahli.asciidoc.confluence.publisher.docker.DockerBasedPublishingIntegrationTest.publish(DockerBasedPublishingIntegrationTest.java:208)
	at org.sahli.asciidoc.confluence.publisher.docker.DockerBasedPublishingIntegrationTest.deleteAllPages(DockerBasedPublishingIntegrationTest.java:50)
```

--- a/asciidoc-confluence-publisher-docker/src/it/java/org/sahli/asciidoc/confluence/publisher/docker/DockerBasedPublishingIntegrationTest.java
+++ b/asciidoc-confluence-publisher-docker/src/it/java/org/sahli/asciidoc/confluence/publisher/docker/DockerBasedPublishingIntegrationTest.java
@@ -16,6 +16,7 @@
 
 package org.sahli.asciidoc.confluence.publisher.docker;
 
+import com.github.dockerjava.api.exception.ConflictException;
 import io.restassured.specification.RequestSpecification;
 import org.junit.Before;
 import org.junit.BeforeClass;
@@ -219,6 +220,8 @@ public class DockerBasedPublishingIntegrationTest {
 
             publisher.start();
             runnable.run();
+        } catch (ConflictException ignored) {
+            // avoid test failures due to issues with already terminated confluence publisher container
         }
     }
 

--- a/asciidoc-confluence-publisher-docker/src/it/resources/testcontainers.properties
+++ b/asciidoc-confluence-publisher-docker/src/it/resources/testcontainers.properties
@@ -1,0 +1,17 @@
+#
+# Copyright 2020 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+checks.disable=true

